### PR TITLE
small repo update -fix null values

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
Jenkins retest please

File: repository/src/main/java/org/jbpm/process/workitem/repository/service/RepoService.java [hunk 1]
```diff
@@ -59,6 +60,9 @@ public RepoService() {
             services = mapper.readValue(jsonInput,
                                         new TypeReference<List<RepoData>>() {
                                         });
+            // remove all nulls from list if any (can happen due to trailing commans in json)
```
Comment (MarianMacik) [inline repository/src/main/java/org/jbpm/process/workitem/repository/service/RepoService.java]: Looks good, just small typo: commans -> commas probably. Of course it's up to you whether you change it :)

Comment (tsurdilo) [reply repository/src/main/java/org/jbpm/process/workitem/repository/service/RepoService.java]: thank you! will update

**Comment by tsurdilo** [comment]
jenkins failed to pull https://repo.maven.apache.org/maven2/com/google/code/findbugs/jsr305/3.0.1/ for some reason. hope wont fail on it again

**Pull request merged**